src/MuTalk-Model/MTParametricMethodMutation.class.st

```diff
@@ -0,0 +1,41 @@
+Class {
+	#name : 'MTParametricMethodMutation',
```

Comment:
Do we really need this class? Why not merge it with its subclass?

Reply:
Ah I see this does not have a subclass... If its just about a parameter, why not adding parameters to the original mutation class then?

Reply:
I can try that